addon/components/confidentiality-sidebar-widget.js

```diff
@@ -22,12 +25,12 @@ export default class ConfidentialitySidebarWidget extends Component {
     return steps.some((step) => step.type === 'operation-step');
   }
 
-  update(transaction) {
+  update = (transaction) => {
```

Comment:
As a personal opinion I would rather have the normal function and then bind it, if you don't want to manually bind you can also add the @action decorator.